src/recommendations/RecommendationsPage.jsx

```diff
@@ -56,7 +56,7 @@ const RecommendationsPage = () => {
   const handleSkip = (e) => {
     e.preventDefault();
     trackSkipButtonClicked(userId);
-    handleSkipRecommendationPage();
+    setTimeout(() => { handleSkipRecommendationPage(); }, LINK_TIMEOUT);
```

Comment:
why we are using settimeout here? Can we not avoid it?

Reply:
we use setTimeout to give a delay before redirecting if we are firing event also